Sources/WebDriver/Session.swift

```diff
@@ -317,7 +317,22 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
-    /// Deletes the current session.
+    /// - Parameter orientation: Orientation the window will flip to {LANDSCAPE|PORTRAIT}
+    public func portraitOrientation(orientation: ScreenOrientation = .portrait) throws {
```

Comment:
We should have a single method taking a parameter. As it is, you could do `portraitOrientation(.landscape)`, which is confusing.
```suggestion
    public func orientation(_ value: ScreenOrientation) throws {
```